go/test/endtoend/cluster/cluster_process.go

```diff
@@ -246,6 +245,11 @@ func (cluster *LocalProcessCluster) StartTopo() (err error) {
 		cluster.VtctlProcess.LogDir = cluster.TmpDirectory
 	}
 
+	if err = cluster.TopoProcess.OpenServer(*topoFlavor, cluster.VtctlProcess.TopoGlobalRoot); err != nil {
```

Comment:
How was this working earlier? If we weren't starting the topo at all..

Reply:
never needed to directly, because everything would happen through the `VtctlProcess`, which makes its own call to `topo.Open`, or one of the clients.